raiden/tests/fixtures/variables.py

```diff
@@ -238,3 +239,13 @@ def raiden_udp_ports(number_of_nodes, port_generator):
 def rest_api_port_number(port_generator):
     """ Unique port for the REST API server. """
     return next(port_generator)
+
+
+@pytest.fixture
+def database_paths(tmpdir, private_keys):
```

Comment:
one small note: in tests you may use the special path `':memory:'` in order to avoid writing to actual files:
```
>>> sqlite3.connect(':memory:')
<sqlite3.Connection at 0x7fddef2ed4b8>
```